gpt_researcher/actions/report_generation.py

```
@@ -222,6 +222,7 @@ async def generate_report(
     custom_prompt: str = "", # This can be any prompt the user chooses with the context
     headers=None,
     prompt_family: type[PromptFamily] | PromptFamily = PromptFamily,
+    available_images: list = None,
```

Ruff (0.14.14)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)